### ISSUE TITLE
segloss function

### ISSUE THREAD

**Issue by Clever-Huskie** [JianghaoWu/SIFA-pytorch#2]
Hallo, I have a question about whether the seg function in your pytorch implementation is somewhat different from the original paper

**Comment by JianghaoWu** [comment]
It is the same, but due to different devices, it is recommended that you use a small learning rate, such as 1 * 10 ^ -5

**Comment by JianghaoWu** [comment]
The provided code is nearly identical to what you need. You can utilize this code as a basis, and kindly acknowledge the implementation from this GitHub repository in your footnotes.
